tests/basic/sharedv4_svc_test.go

```diff
@@ -63,8 +63,15 @@ var _ = Describe("{NFSServerFailover}", func() {
 				for name := range applicationScaleUpMap {
 					applicationScaleUpMap[name] = int32(0)
 				}
-				err = Inst().S.ScaleApplication(ctx, applicationScaleUpMap)
-				Expect(err).NotTo(HaveOccurred())
+				var scaleErr error
+				for i := 0; i < 3; i++ {
+					scaleErr = Inst().S.ScaleApplication(ctx, applicationScaleUpMap)
```

Comment:
We should fix this in K8s) ScaleApplication() function. Catch the relevant k8s error and retry. That way, every job benefits from this fix.

Reply:
good point, will do

Reply:
I used wait.ExponentialBackoff in the px change. However, that module is not in torpedo, so I didn't use it here. Let me know if you think it is worth adding that module to torpedo.

Reply:
LGTM